electionleaflets/settings/base.py

```diff
@@ -110,6 +111,13 @@
     'django_extensions',
 ] + LEAFLET_APPS
 
+if environ.get('SENTRY_DSN', None):
+    INSTALLED_APPS.append('raven.contrib.django.raven_compat')
+    RAVEN_CONFIG = {
+        'dsn': environ.get('SENTRY_DSN'),
+        'environment': environ.get('SENTRY_ENVIRONMENT', 'production')
```

Comment:
It's in the main Python docs https://docs.sentry.io/clients/python/advanced/#client-arguments